packages/frontend/src/components/common/modal/DashboardUpdateModal.tsx

```diff
@@ -31,69 +31,73 @@ const DashboardUpdateModal: FC<DashboardUpdateModalProps> = ({
     const { mutateAsync, isLoading: isUpdating } = useUpdateDashboard(uuid);
 
     const form = useForm<FormState>({
-        mode: 'onChange',
-        defaultValues: {
-            name: dashboard?.name,
-            description: dashboard?.description,
+        initialValues: {
+            name: '',
+            description: '',
         },
     });
 
+    const { setValues } = form;
+
+    useEffect(() => {
+        if (!dashboard) return;
+
+        setValues({
+            name: dashboard.name,
+            description: dashboard.description ?? '',
+        });
+    }, [dashboard, setValues]);
+
     if (isLoading || !dashboard) {
         return null;
     }
 
-    const handleConfirm = async (data: FormState) => {
+    const handleConfirm = form.onSubmit(async (data) => {
         await mutateAsync({
             name: data.name,
             description: data.description,
         });
         onConfirm?.();
-    };
+    });
 
     return (
-        <Dialog lazy title="Update Dashboard" icon="control" {...modalProps}>
-            <Form
-                title="Update Dashboard"
-                methods={form}
-                onSubmit={handleConfirm}
-            >
-                <DialogBody>
-                    <Input
+        <Modal
+            title={<Title order={4}>Update Dashboard</Title>}
+            {...modalProps}
+        >
+            <form title="Update Dashboard" onSubmit={handleConfirm}>
+                <Stack spacing="lg" pt="sm">
+                    <TextInput
                         label="Enter a memorable name for your dashboard"
-                        name="name"
+                        required
                         placeholder="eg. KPI Dashboards"
                         disabled={isUpdating}
-                        rules={{ required: 'Name field is required' }}
```

Comment:
now that we don't have `rules` will the validation still work?

Reply:
yes because it's `required`! 👍